lib/parser.js

```diff
@@ -59,15 +59,14 @@ class Parser extends EventEmitter {
       entry.name = Object.keys(entry.value)[0]
       entry.value = entry.value[entry.name]
 
-      if (entry.value instanceof utils.NodeFunction) {
-        switch (property.key.name) {
-          case 'computed':
-            entry.dependencies = utils.getDependencies(entry.value, this.source.script)
-            break
-
-          default:
-            entry.params = entry.value.params
+      if (property.key.name === 'computed') {
+        if (entry.value instanceof utils.NodeFunction) {
+          entry.dependencies = utils.getDependencies(entry.value, this.source.script)
+        } else if (entry.value instanceof Object && entry.value.get instanceof utils.NodeFunction) {
```

Comment:
Finally I think the `else` block is require to set `entry.dependencie = []`.
What do you think?

Reply:
Hi, ok I will change it to return an empty array and change the test.

Reply:
Done :-)

Reply:
Great!!!
Thank you @pschmiedel for your PR. 
I'll release it soon.

Reply:
Thanks a lot for the quick response!